README.rst

```diff
@@ -1,76 +1,51 @@
-|Build Status| |Codecov| |license|
```

Comment:
I think we'll want to keep the `license` here.

Assumption: we'll need to add a `LICENSE` file at some point. Not sure if there is a SOP for adding one to be honest though.

Reply:
yes and we should work with @nedbat to determine the right license to use

Reply:
There is a license included from the template: https://github.com/edx/frontend-app-skills/blob/main/LICENSE

This appears to be the same for other MFEs. I can plan to link this one for now

Reply:
@MaxFrank13 It should be in this repo, AFAIK, we shouldn't link to one in another repo. Let's just add the file to if possible.

Reply:
errr, may have read this wrong, afterwards I realized there is a LICENSE file already in this repo and that we were talking about the file that already exists, not linking to a LICENSE file in another repo... ignore me :)

Reply:
AGPL is a fine choice.  We have guidance on how to choose between AGPL and Apache: https://openedx.atlassian.net/wiki/spaces/OPEN/pages/13959332/Licensing